subsys/caf/modules/power_manager.c

```diff
@@ -222,6 +267,33 @@ static bool event_handler(const struct event_header *eh)
 		EVENT_SUBMIT(event);
 		return false;
 	}
+#if IS_ENABLED(CONFIG_CAF_POWER_MANAGER_RUNTIME_TIMEOUT_SET)
+	if (is_power_manager_configure_timeout_event(eh)) {
+		int err;
+		const struct power_manager_configure_timeout_event *event =
+			cast_power_manager_configure_timeout_event(eh);
+		int timeout_interval;
+
+		memcpy(&timeout_interval, &(event->timeout_seconds), sizeof(int));
```

Comment:
`memcpy(&timeout_interval, &event->timeout_seconds, sizeof(timeout_interval));`

Use variable name instead of type when using sizeof. Makes code easier to maintain and has lower chance of bugs as size comes directly from destination.

You don't need a brackets after `&` in the second argument. Access of structure member has a documented precedence over address of.

Reply:
fix sizeof argument in other places